mitodl/open-discussions
# add validation for extremely long text posts

#### Pre-Flight checklist



#### What are the relevant tickets?


closes #1469 

#### What's this PR do?

adds validation that the text post body isn't longer than the backend allows.

#### How should this be manually tested?

Go to the post create form, and try to create a text post. you should be able to go forward with creating a reasonably-sized text post no problem. then, try making a reeeeeally long text post. You can do this in the JavaScript console to get a nice long string:

```js
const longString = Array(40000).fill().map(() => "a").join("")
```

If the string is 40000 characters long you should be able to save it, but you should get a validation error for anything over that (see screenshot below).

#### Screenshots (if appropriate)

![waaaaaytoolongmessage](https://user-images.githubusercontent.com/6207644/48430860-4667da80-e73e-11e8-9b52-e552561da673.png)
